cassandra-schema.cql

```diff
@@ -1093,9 +1093,12 @@ CREATE TABLE brig_test.invitee_info (
     AND speculative_retry = '99PERCENTILE';
 
 CREATE TABLE brig_test.nonce (
-    key text PRIMARY KEY,
-    nonce uuid
-) WITH bloom_filter_fp_chance = 0.01
+    user uuid,
```

Comment:
Good catch, thanks!

I was going to say that we wanted to avoid typing the key, but I can't think of a hypothetical use case where asking for a `UserId` doesn't make sense.  So, good, I think.